bootstrap/kubeadm/config/default/kustomization.yaml

```diff
@@ -9,8 +9,8 @@ namespace: capi-kubeadm-bootstrap-system
 namePrefix: capi-kubeadm-bootstrap-
 
 # Labels to add to all resources and selectors.
-#commonLabels:
-#  someName: someValue
+commonLabels:
+  cluster.x-k8s.io/provider: "kubeadm-bootstrap"
```

Comment:
Can we flip this to `bootstrap-kubeadm`?

Reply:
cc @fabriziopandini

Reply:
I'm ok, but then also `controlplane-kubeadm`; 
This change should be reflected in clusterctl as weel (out of scope of this PR) and in clusterct this will looks like `clusterct init --bootstrap bootstrap-kubeadm --controlplane controlplane-kubeadm`

Reply:
@fabriziopandini I'll flip the label value. If you want to create a separate issue for the clusterctl flag values (stuff you mentioned above), feel free to assign that to me. I can work on that.